src/index.css

```diff
@@ -10,4 +10,17 @@ body {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
-}
\ No newline at end of file
+}
+
+.ui.header {
+  text-align: center;
+  padding-top: 20px;
```

Comment:
its a very small point but percentages or similar rather than defined sizes result in the site being responsive on mobile devices